### ISSUE TITLE
hi, can not get result from paper, CIFAR10 IPC=10. acc of your paper is 53.5,but I only get 49.97

### ISSUE THREAD

**Issue by Luo-Zhongwei** [shaoshitong/G_VBSM_Dataset_Condensation#3]
`======= FKD: dataset info ======
   2 path: /home/zhanglf/lzw/code/GVBSM_spkd/Branch_CIFAR_10/relabel/FKD_cutmix_fp16FKD_IPC_10
   3 num img: 100
   4 batch size: 100
   5 max epoch: 1000
   6 ================================
   7 Files already downloaded and verified
   8 load data successfully
   9 => loading student model 'ResNet18'`

`Epoch: 999
TRAIN Iter 999: lr = 0.000000,  loss = 0.221218,        Top-1 err = 20.000000,  Top-5 err = 4.000000,   train_time = 19.630981
TEST Iter 999: loss = 1.362584, Top-1 err = 50.030000,  Top-5 err = 5.950000,   val_time = 98.427372
wandb:                                                                                
wandb: 
wandb: Run history:
wandb:  train/Top1 ▃▂▆▅▆▄▂▆▅▇▂▃▂▆█▁▆▅█▅██▆▇▄▆▆▇▁▇▅▆▇▆▇▇▅▇█▂
wandb:  train/Top5 ▅▄▇▆▇▅▃▆▇▇▄▅▂▇█▁█▆█▆██▇█▆▇▇▇▂█▆▇▇▇█▇▆▇█▂
wandb: train/epoch ▁▁▁▂▂▂▂▂▂▃▃▃▃▃▄▄▄▄▄▄▅▅▅▅▅▅▆▆▆▆▆▇▇▇▇▇▇███
wandb:  train/loss █▅▃▃▃▃▄▃▃▂▃▂▃▂▂▄▂▂▁▂▁▁▂▁▂▂▁▁▃▁▂▂▁▂▁▁▂▁▁▃
wandb:    train/lr ███████▇▇▇▇▇▆▆▆▆▆▅▅▅▄▄▄▄▃▃▃▃▂▂▂▂▂▁▁▁▁▁▁▁
wandb:   val/epoch ▁▁▁▁▂▂▂▂▂▃▃▃▃▃▃▄▄▄▄▄▅▅▅▅▅▅▆▆▆▆▆▇▇▇▇▇▇███
wandb:    val/loss █▃▂▂▂▂▁▂▂▁▁▁▂▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁
wandb:    val/top1 ▁▄▅▆▆▆▇▇▆▇▇▇▆▇▇▇▇█▇█████▇███████████████
wandb:    val/top5 ▁▆▇▇▇▇█▇█▇██▇███████████████████████████
wandb: 
wandb: Run summary:
wandb:  train/Top1 80.0
wandb:  train/Top5 96.0
wandb: train/epoch 999
wandb:  train/loss 0.22122
wandb:    train/lr 0.0
wandb:   val/epoch 999
wandb:    val/loss 1.36258
wandb:    val/top1 49.97
wandb:    val/top5 94.05
wandb: 
wandb: 🚀 View run playful-sun-9 at: https://wandb.ai/485434977-shang-hai-jiao-tong/final_rn18_fkd/runs/9d0uywbl
wandb: ⭐️ View project at: https://wandb.ai/485434977-shang-hai-jiao-tong/final_rn18_fkd
wandb: Synced 6 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20240903_181225-9d0uywbl/logs`

**Comment by Zhongwei-Luo** [comment]
**val/top1 49.97**

**Comment by Zhongwei-Luo** [comment]
hi

**Comment by shaoshitong** [comment]
Thank you for your interest in our work! I apologize for taking so long to get back to you, as I have been very busy for a while now. This performance should be reproducible, perhaps because there are fluctuations on CIFAR-10 causing some differences in performance. If you really can't reproduce it, we give some better quality hyperparameter configurations on work[EDC](https://arxiv.org/pdf/2404.13733) that you can try:

- batch szie=25
- scheduler=multistep, adjust the learning rate down to the original 0.5 at epochs [800,900,950], respectively
- ce-weight changed to 0.025

These if still not work, you can refer to the document [direct_train](https://github.com/shaoshitong/EDC/blob/main/Branch_CIFAR_10/train/direct_train.py) using direct_train form (which introduces RandAugmentation for better results). In addition, we have designed Soft Category-Aware Matching and Flatness Regularization in EDC, which might be helpful for further performance improvement.
